client/src/main/java/com/tigrisdata/db/client/StandardTigrisDatabase.java

```diff
@@ -147,6 +150,21 @@ public DatabaseDescription describe() throws TigrisException {
     return TypeConverter.toDatabaseDescription(response);
   }
 
+  @Override
+  public Iterator<StreamEvent> stream() throws TigrisException {
+    Api.StreamRequest streamRequest = Api.StreamRequest.newBuilder().setDb(db).build();
+    Iterator<Api.StreamResponse> streamResponseIterator = blockingStub.stream(streamRequest);
+    Function<Api.StreamResponse, StreamEvent> converter =
+        streamResponse -> {
+          try {
+            return StreamEvent.from(streamResponse.getEvent(), objectMapper);
+          } catch (IOException e) {
+            throw new IllegalArgumentException("Failed to convert event data to JSON", e);
```

Comment:
Please move the error message to Messages constant holder. 

re: IllegalArgumentException - In my mind this should be thrown on client errors. here the argument is supplied to this library from server. Should we call it just TigrisException or IllegalStateException? wdyt?

Reply:
I'd agree that IllegalArgumentException doesn't make sense here, I was just following the pattern again